crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -1,12 +1,16 @@
 use crate::{
-    environment_map, prepass, render::morph::MorphIndices, EnvironmentMapLight, FogMeta,
-    GlobalLightMeta, GpuFog, GpuLights, GpuPointLights, LightMeta, MaterialBindGroupId,
-    NotShadowCaster, NotShadowReceiver, PreviousGlobalTransform,
+    environment_map, prepass,
+    render::{
+        morph::{no_automatic_morph_batching, MorphIndices},
+        skin::no_automatic_skin_batching,
```

Comment:
Could those two imports be moved at line 54? this would avoid some merge conflicts. on #9902 :sweat_smile:

Reply:
Sure!